frontend/src/tbpro/elements/TextInput.vue

```diff
@@ -25,21 +29,28 @@ interface Props {
   remoteError?: string;
   type?: string;
   placeholder?: string;
+  prefix?: string; // A prefix shows up at the start of the input field and moves the actual input to the right.
   required?: boolean;
   disabled?: boolean;
 }
-withDefaults(defineProps<Props>(), {
+const props = withDefaults(defineProps<Props>(), {
   type: 'text',
   help: null,
   remoteError: null,
   placeholder: '',
+  prefix: '',
```

Comment:
Can we set this to null? (Placeholder should also be null but that's my fault.)

Reply:
Of course. Haha sorry, sometimes I just duplicate properties as they are when adding new ones :D